### ISSUE TITLE
Real life working example

### ISSUE THREAD

**Issue by mparpaillon** [mathieudutour/cordova-plugin-hotpushes#10]
Hi,

I'd like to update some of my contents remotely (assets and fixtures) but I think I'm missing something.

For a replace I have to create a zip, but what about the merge scenario.
The files need to be on the server, right ? Where ? Zipped or not ? Does it have to follow the exact same tree as in the app ? 
Then when it's downloaded, will it automatically replace all the local files ?
Should the file path be the same between the local and the remote version.json files ?

In short, can you give a real life working example with server side and client side.
A basic Cordova app sample for instance.

I'm ready to help if needed.
Thanks a lot
